dev-docs/examples/kaltura-example.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: player_example
+title: Kaltura Player Integration
+description: Show a prebid ad in a Kaltura player
+
+top_nav_section: dev_docs
+nav_section: quick-start
+
+hide: true
+beta: true
+
+about:
+- Using invokeVideoPlayer to set up and access the Kaltura player instance.
+- Using the dynamic embed integration method with the Kaltura player.
+
+player_notes:
+- In Kaltura studio, enable the advertising plugin you want to use. For this demo we'll be using the VAST 3.0 plugin.
+- We'll be using the <b>Dynamic Embed</b> method to add the player to the page. We need to customize the player on the page, so the auto embed method will not work.
+- To find the Dynamic Embed code for your player, go to the content tab and select the 'Preview and Embed' action. Click 'Show Advanced Options' to see more embedding options, and select 'Dynamic Embed' from the dropdown menu.
+
+jsfiddle_link: jsfiddle.net/shirleylberry/17vap1ro/embedded/html/
+demo_link: video-demo.appnexus.com/pbjs/kaltura-prebid/kaltura-demo.html
```

Comment:
this link is 404'ing:

http://video-demo.appnexus.com/pbjs/kaltura-prebid/kaltura-demo.html